### DISCUSSION TITLE
There is not any test file in tensorboard  log directory

### DISCUSSION THREAD

**Discussion by saeedshahi** [deepmodeling/deepmd-kit#1748] [Q&A] [answered]
Hi,
I try to use tensorboard results but there is just this file log/train/events.out.tfevents.1654622828.node399 and log/test directory is empty.
I have both training_data and validation_data in the training section. Also, I can see both validation and training results in lcurve.out but nothing in log/test directory.
Should I add something in my input.json file to obtain the tensorboard test file?

**Comment by njzjz** [comment, marked as answer]
Do you enable TensorBoard option?

**Reply by saeedshahi** [reply]
if you mean this "tensorboard":  true,  yes I did

**Reply by njzjz** [reply]
I reviewed the code and found validation results had not been supported yet. So if you found nothing, that's the expected behavior.

**Reply by xyh01** [reply]
> I reviewed the code and found validation results had not been supported yet. So if you found nothing, that's the expected behavior.

Do you plan to realize this function?

**Reply by njzjz** [reply]
> Do you plan to realize this function?

No, but contribution is welcome.

**Comment by xyh01** [comment]
Ok, thanks.



---Original---
From: "Jinzhe ***@***.***&gt;
Date: Sat, May 6, 2023 04:17 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [deepmodeling/deepmd-kit] There is not any test file intensorboard  log directory (Discussion #1748)




  
Do you plan to realize this function?
  
No, but contribution is welcome.
